doc/relevancy.md

```diff
@@ -0,0 +1,383 @@
+# Relevancy
+
+This document explains how documents are ranked when you perform a search
+with results sorted by relevancy.
+
+## Contents
+
+1. What is relevancy?
+2. Impacts on relevancy
+3. Known issues / Possible problems with queries and relevance
+
+## What is relevancy?
+
+A list of documents returned by Search API will include an `es_score` on every document.
+
+```ruby
+# Response for a search for 'Harry Potter'
+[
+  { title: "Harry Potter", es_score: 1 },
+  { title: "Harry Kane", es_score: 0.5 },
+  { title: "Ron Weasley", es_score: 0.05 }
+]
+```
+
+The `es_score` value represents the rank of the result or, more generally, how relevant we think a result is to your query.
```

Comment:
It's the score, not the rank.  Ranks would be 1 for the first result, 2 for the second, etc.

> The `es_score` value represents the rank of the result or, more generally, how relevant we think a result is to your query.

is repetitive, so maybe

> The `es_score` value is used for ranking results and represents how relevant we think a result is to your query.